slick/src/main/scala/slick/jdbc/OracleProfile.scala

```diff
@@ -404,3 +556,180 @@ object OracleProfile extends OracleProfile {
     case class AutoIncTriggerName(name: String) extends ColumnOption[Nothing]
   }
 }
+
+
+/* The following code will be used when the native implementation of
```

Comment:
what does this comment mean? "native implementation of ... are natively implemented"

Reply:
That comment was left over from one of the intermediate commits. Removed.